internal/pkg/otel/samples/darwin/logs_hostmetrics.yml

```diff
@@ -0,0 +1,36 @@
+receivers:
```

Comment:
This is the same file as the above `logs_host_metrics.yml`, right? :slightly_smiling_face:

Reply:
yeah i wanted to do resource enrichment but i will take it out